services/movie.py

```diff
@@ -0,0 +1,36 @@
+from django.db.models import QuerySet
+
+from db.models import Movie
+
+
+def get_movies(genres_ids: list[int] = None,
+               actors_ids: list[int] = None
+               ) -> QuerySet | Movie:
```

Comment:
The return type hint `QuerySet | Movie` is incorrect for the `get_movies` function. Since the function always returns a QuerySet, the correct type hint should be `QuerySet[Movie]`.